README.md

```diff
@@ -226,33 +241,34 @@ mode: 'cloud'
 Or you can just run: `./ftw run --cloud`
 
 ## How log parsing works
-The log output from your WAF is parsed and compared to the expected output.
-The problem with log files is that they aren't updated in real time, e.g. because the
-web server / WAF has an internal buffer, or because there's some `fsync` magic involved).
-To make log parsing consistent and guarantee that we will see output when we need it,
-go-ftw uses "log markers". In essence, unique log entries are written _before_ and _after_
-every test stage. go-ftw can then search for these markers.
+
+The WAF's log file with the alert messages is parsed and compared to the expected output defined in the unit test under `log_contains` or `no_log_contains`.
+
+The problem with log files is that `go-ftw` is very, very fast and the log files are not updated in real time. Frequently, the
+web server / WAF is not syncing the file fast enough. That results in a situation where `go-ftw` won't find the log messages it has triggered.
+
+To make log parsing consistent and guarantee that we will see output when we need it, `go-ftw` will send a request that is meant to write a marker into the log file before the individual test and another marker after the individual test.
+
+If `go-ftw` does not see the finishing marker after executing the request, it will send the marker request again until the webserver is forced to write the log file to the disk and the marker can be found.
 
 The [container images for Core Rule Set](https://github.com/coreruleset/modsecurity-crs-docker) can be configured to write these marker log lines by setting
-the `CRS_ENABLE_TEST_MARKER` environment variable. If you are testing a different WAF
-you will need to instrument it with the same idea (unless you are using "cloud mode").
+the `CRS_ENABLE_TEST_MARKER` environment variable. If you are testing a different test setup, you will need to instrument it with a rule that generated the marker in the log file via a rule alert (unless you are using "cloud mode").
+
 The rule for CRS looks like this:
+
 ```
 # Write the value from the X-CRS-Test header as a marker to the log
 SecRule REQUEST_HEADERS:X-CRS-Test "@rx ^.*$" \
-  "id:999999,\
+  "id:900910,\
```

Comment:
If you change it here, you need probably to change it all over the place. In particular, [here](https://github.com/coreruleset/modsecurity-crs-docker/blob/b0603c1832a38eb578eefce462218f367af7c88e/src/opt/modsecurity/activate-rules.sh#L133)